src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/typebodypadding/Example5.java

```diff
@@ -0,0 +1,18 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="TypeBodyPadding">
+      <property name="allowEmpty" value="false"/>
+    </module>
+  </module>
+</module>
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.typebodypadding;
+
+// xdoc section -- start
+public class Example5 { // violation 'A blank line is required after the opening'
+}
```

Comment:
This looks weird, crazy class by design.
It looks weird that this property is to cover this crazy code.
What you trying to cover are cases like:

```
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.TYPE)          
public @interface MyMarkerAnnotation {}
```
https://www.baeldung.com/java-custom-annotation#1-class-level-annotation-example

Please add inner annotation declaration to all examples.
Example5 should be same by code (not a comments) as all others.